doc/classes/RandomNumberGenerator.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="RandomNumberGenerator" inherits="Reference" category="Core" version="3.1">
+	<brief_description>
+		A class for generation pseudo-random numbers.
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<demos>
+	</demos>
+	<methods>
+        <method name="randi">
+            <return type="int">
```

Comment:
You seem to be mixing tab indentation and space indentation within this file.

Reply:
Yeah, I tried to fix it but seems to be not completly